src/dms/clis/EdFi.DataManagementService.SchemaGenerator.Tests.Unit/Engines/Mssql/__snapshots__/MssqlFlatteningMetadataComprehensiveTests.ValidateNestedCommonCollections_GeneratesMultilevelChildTables.snap

```diff
@@ -0,0 +1,255 @@
+        base.Id,
+        base.Document_Id,
+        base.Document_PartitionKey,
+        base.CreateDate,
+        base.LastModifiedDate,
+        base.ChangeVersion,
+        base.StudentId
+    FROM [dms].[edfi_StudentEducationOrganizationAssociation] base');
+    PRINT 'View StudentEducationOrganizationAssociation_View created.';
+END
+ELSE
+BEGIN
+    EXEC('ALTER VIEW [dms].[StudentEducationOrganizationAssociation_View] AS
+    SELECT
+        base.Id,
+        base.Document_Id,
+        base.Document_PartitionKey,
+        base.CreateDate,
+        base.LastModifiedDate,
+        base.ChangeVersion,
+        base.StudentId
+    FROM [dms].[edfi_StudentEducationOrganizationAssociation] base');
+    PRINT 'View StudentEducationOrganizationAssociation_View altered.';
+END
+GO
+
+
+
+-- Natural Key Resolution View for StudentEducationOrganizationAssociationAddress_View
+IF NOT EXISTS (SELECT * FROM sys.views WHERE name = 'StudentEducationOrganizationAssociationAddress_View' AND SCHEMA_NAME(schema_id) = 'dms')
+BEGIN
+    EXEC('CREATE VIEW [dms].[StudentEducationOrganizationAssociationAddress_View] AS
+    SELECT
+        base.Id,
+        base.Document_Id,
+        base.Document_PartitionKey,
+        base.CreateDate,
+        base.LastModifiedDate,
+        base.ChangeVersion,
+        base.StudentEducationOrganizationAssociation_Id,
+        parent.StudentId AS StudentEducationOrganizationAssociation_StudentId,
+        base.StreetNumberName
+    FROM [dms].[edfi_StudentEducationOrganizationAssociationAddress] base
+    INNER JOIN [dms].[edfi_StudentEducationOrganizationAssociation] parent ON base.StudentEducationOrganizationAssociation_Id = parent.Id');
+    PRINT 'View StudentEducationOrganizationAssociationAddress_View created.';
+END
+ELSE
+BEGIN
+    EXEC('ALTER VIEW [dms].[StudentEducationOrganizationAssociationAddress_View] AS
+    SELECT
+        base.Id,
+        base.Document_Id,
+        base.Document_PartitionKey,
+        base.CreateDate,
+        base.LastModifiedDate,
+        base.ChangeVersion,
+        base.StudentEducationOrganizationAssociation_Id,
+        parent.StudentId AS StudentEducationOrganizationAssociation_StudentId,
+        base.StreetNumberName
+    FROM [dms].[edfi_StudentEducationOrganizationAssociationAddress] base
+    INNER JOIN [dms].[edfi_StudentEducationOrganizationAssociation] parent ON base.StudentEducationOrganizationAssociation_Id = parent.Id');
+    PRINT 'View StudentEducationOrganizationAssociationAddress_View altered.';
+END
+GO
+
+
+
+-- Natural Key Resolution View for StudentEducationOrganizationAssociationAddressPeriod_View
+IF NOT EXISTS (SELECT * FROM sys.views WHERE name = 'StudentEducationOrganizationAssociationAddressPeriod_View' AND SCHEMA_NAME(schema_id) = 'dms')
+BEGIN
+    EXEC('CREATE VIEW [dms].[StudentEducationOrganizationAssociationAddressPeriod_View] AS
+    SELECT
+        base.Id,
+        base.Document_Id,
+        base.Document_PartitionKey,
+        base.CreateDate,
```

Comment:
Check the ODS/API tables, I'm not sure child tables have CreateDate, LastModifiedDate or ChangeVersion.